auslib/AUS.py

```diff
@@ -52,9 +52,23 @@ def __init__(self):
         self.rand = functools.partial(randint, 0, 99)
         self.log = logging.getLogger(self.__class__.__name__)
 
+    def get_emergency_shutoff(self, product, channel):
```

Comment:
Will this handle fallback channels as well? Eg: shutoffs for "release" should apply when the updateQuery has a channel of "release-cck-foo".

Reply:
Actually not. Can I consider the database documention to implement this?

> The channel has special handling to try "falling back" to a simpler channel, for example a request with release-cck-foo will also consider rules for 'release'. This only applies to channels containing '-cck-'.

In other words, the portion of channel name before '-cck-' sequence is the fallback channel.

Reply:
Yup, that's exactly right.

Reply:
Oh! Thanks for response, I realized now that the answer for my question is at line 44 in that file!